charts/fluent-operator/Chart.yaml

```diff
@@ -1,31 +1,26 @@
 apiVersion: v2
 name: fluent-operator
-description: A Helm chart for Kubernetes
-
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
-# type: application
-
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.1.0
-
-# This is the version number of the application being deployed. This version number should be
-# incremented each time you make changes to the application. Versions are not expected to
-# follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: 3.1.0
-
+description: Fluent Operator provides great flexibility in building a logging layer based on Fluent Bit and Fluentd.
```

Comment:
Updated to match the contents of https://github.com/fluent/helm-charts/blob/main/charts/fluent-operator/Chart.yaml